fdbcli/fdbcli.actor.cpp

```diff
@@ -3795,29 +3789,8 @@ ACTOR Future<int> cli(CLIOptions opt, LineNoise* plinenoise) {
 				}
 
 				if (tokencmp(tokens[0], "consistencycheck")) {
-					getTransaction(db, tr, options, intrans);
-					tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
-					tr->setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
-					tr->setOption(FDBTransactionOptions::LOCK_AWARE);
-					if (tokens.size() == 1) {
-						state Future<Optional<Standalone<StringRef>>> ccSuspendSettingFuture =
-						    tr->get(fdbShouldConsistencyCheckBeSuspended);
-						wait(makeInterruptable(success(ccSuspendSettingFuture)));
-						bool ccSuspendSetting =
-						    ccSuspendSettingFuture.get().present()
-						        ? BinaryReader::fromStringRef<bool>(ccSuspendSettingFuture.get().get(), Unversioned())
-						        : false;
-						printf("ConsistencyCheck is %s\n", ccSuspendSetting ? "off" : "on");
-					} else if (tokens.size() == 2 && tokencmp(tokens[1], "off")) {
-						tr->set(fdbShouldConsistencyCheckBeSuspended, BinaryWriter::toValue(true, Unversioned()));
-						wait(commitTransaction(tr));
-					} else if (tokens.size() == 2 && tokencmp(tokens[1], "on")) {
-						tr->set(fdbShouldConsistencyCheckBeSuspended, BinaryWriter::toValue(false, Unversioned()));
-						wait(commitTransaction(tr));
-					} else {
-						printUsage(tokens[0]);
-						is_error = true;
-					}
+					bool _result = wait(consistencycheckCommand(db2, tokens));
```

Comment:
This is ignoring `options` and `intrans` now. This is technically a behavior change, so we should make sure that's acceptable.

Probably ignoring `intrans` is reasonable, as it doesn't seem like this should need to be transactional with other fdbcli operations and I'm not sure it makes sense that invoking this command should commit your other transaction mutations.

It's possible that there could be options that one would want to set on this transaction, if not now then in the future, so I'm less certain about those.

Reply:
Thanks for finding this.
I forgot these two options here.
I will fix this.

Reply:
Added the code to reuse the underlying transaction object to make it consistent.
Finally, when the refactoring is done, 
we will have `tr` as `Reference<ITransaction>` and `db` as `Reference<IDatabase>`.
But since the refactoring all at a time is painful,
we now use the temporary `tr2` and `db2`.